### DISCUSSION TITLE
Use local path for image covers

### DISCUSSION THREAD

**Discussion by g0kuvonlange** [xbapps/xbvr#1307] [Ideas]
I have a couple of custom edited movies with my own custom cover image that is stored alongside the video file and script.

I've tried referencing a local image path that I would like to use as cover image. Unfortunately no matter what path I've tried, it didn't work. So I assumed it's working strictly by scraping the image url's on the web.

Support for locally stored images would be much appreciated :)

**Comment by Charlotte-br560** [comment]
It sounds like you're looking to use locally stored images for cover art. Currently, the system seems to only support scraping image URLs from the web. However, adding support for locally stored images is a great idea and would certainly enhance user customization options.
In the meantime, if you're interested, you might want to check out [Crawlbase ](https://crawlbase.com/)as an alternative solution for your needs.

**Reply by theRealKLH** [reply]
you can actually use local files since a couple of releases now (#1457 ).
